src/window.rs

```diff
@@ -76,7 +76,7 @@ impl Window {
     }
     /// The window is focused.
     pub fn is_focused(&self) -> bool {
-        self.impl_window.is_focused
+        self.impl_window.is_focused()
```

Comment:
If we don't get the value from a function, we get the focused app when the `Window` has been built. now we get if the value is focused at the time of checking instead of a static value